docs/calico.md

```diff
@@ -50,6 +50,16 @@ or for versions prior *v1.0.0*:
 calicoctl endpoint show --detail
 ```
 
+##### Optionnal : Define network backend
+
+In some cases you may want to define Calico network backend. Allowed values are 'bird', 'gobgp' or 'none'. Bird is a default value.
```

Comment:
But since we've removed default value from Kargo, we can also remove this sentence:
>Bird is a default value.